gateway-js/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ This CHANGELOG pertains only to Apollo Federation packages in the 2.x range. The
  - Fix fragment reuse in subgraph fetches [PR #1911](https://github.com/apollographql/federation/pull/1911).
  - Allow passing a custom `fetcher` [PR 1997#](https://github.com/apollographql/federation/pull/1997).
   - __UNBREAKING__: Previous 2.1.0 alphas removed the custom fetcher for Apollo Uplink. This re-adds that parameter, and requires the fetcher to have the `AbortSignal` interface https://fetch.spec.whatwg.org/#requestinit.
+- The method `RemoteGraphQLDataSource.errorFromResponse` now returns a `GraphQLError` (as defined by `graphql`) rather than an `ApolloError` (as defined by `apollo-server-errors`). [PR #2028](https://github.com/apollographql/federation/pull/2028)
```

Comment:
**suggestion** Would you mind adding a `__BREAKING__` sub-bullet to this about the change in return type? I agree it's _probably_ low-impact but might as well be explicit.

Reply:
Done

Reply:
also tweaked some formatting in some of the lines above